scss/components/_activity-filter.scss

```diff
@@ -158,3 +158,29 @@
     margin-top: 5px;
   }
 }
+
+@media (max-width: 1420px) {
+  .civicase__case-details-panel:not(.civicase__case-details-panel--focused) {
+    .civicase__activity-filter {
+      padding: 16px 10px;
+    }
+
+    .civicase__activity-filter__timeline {
+      width: 120px !important;
```

Comment:
Do you think, we need to document these widths in the variable?
 Can we add a basic documentation in the scss file only. So that this helps another dev to understand why have we did this.

Reply:
> Do you think, we need to document these widths in the variable?
Again(same as below), I feel this should not go to a variable.

Variable should be created if a value is used more than once, or the value is not self explanatory. 
In this case both cases are false. Because from the code 
1. This is not used more than once,
2. Its clear that we want to fix the width of `civicase__activity-filter__timeline` when screen size is under 1420px.

> Can we add a basic documentation in the scss file only
I feel this is unnecessary comments that we will be adding. Documentation(specially in the code) should only be made if absolutely necessary or if code is not self explanatory. But in this case, its just a simple width set under a media query, so its seems clear(to me) that the intention is to limit the width.

Let me know if my logic is not satisfactory.